packages/blocks/src/page-block/edgeless/toolbar/brush-tool/brush-tool-button.ts

```diff
@@ -101,8 +106,11 @@ export class EdgelessBrushToolButton extends LitElement {
 
     return html`
       <edgeless-tool-icon-button
-        .tooltip=${getTooltipWithShortcut('Pen', 'P')}
+        .tooltip=${this._popperShow
+          ? undefined
+          : getTooltipWithShortcut('Pen', 'P')}
         .active=${type === 'brush'}
+        .testId=${'brush'}
```

Comment:
It is recommended to use locatorEdgelessToolButton to locate the tool button, without the need to add a new testId.